rfcs/text/0030-linux-event-model.md

```diff
@@ -1736,6 +2063,30 @@ Stage 2: Identifies scope of impact of changes. Are breaking changes required? S
 The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
 -->
 
+### Ingestion mechanisms
+
+The Adaptive workload protections (AWP) sensor team is working hard on integrating these new ECS fields into the endpoint agent codebase. The existing kprobe sensor tech will be upgraded to support much of the new widened process context defined in this RFC. At the same time, eBPF sensor technology is being worked on and will eventually be the go to method for building these process events.
+
```

Comment:
"kprobe sensor tech" -> "perf/tracefs kprobe sensor technology".

"support much of the new" - do we know of any specific fields in this RFC that cannot be supported? Ideally Endpoint users don't have to worry about which underlying tech is used to be build these events.

Reply:
cc @mattnite

Reply:
None of the fields so far implemented are not supported by the kprobes. It should be entirely transparent to the user which mechanism is gathering the data.